subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/JavaUtilLoggingSystemIntegrationTest.groovy

```diff
@@ -88,6 +88,33 @@ class JavaUtilLoggingSystemIntegrationTest extends AbstractIntegrationSpec {
     }
 
     @NotYetImplemented
```

Comment:
Yeah, it's still not implemented. At the moment [all of our log levels map to Java util log level `FINE`](https://github.com/gradle/gradle/blob/master/subprojects/logging/src/main/java/org/gradle/internal/logging/source/JavaUtilLoggingSystem.java#L42-L47). @jjohannes made that change and I pinged him about it.